crates/bdk/src/wallet/mod.rs

```diff
@@ -1986,6 +1986,14 @@ impl<D> Wallet<D> {
     pub fn local_chain(&self) -> &LocalChain {
         &self.chain
     }
+
+    /// Get the [`KeychainKind`] and derivation index for a given [`Script`].
+    ///
+    /// This will return `None` if the script is not in the wallet.
+    /// TODO: Generalize this implementation once we make the wallet generic over K
```

Comment:
I think this TODO should be in `//`, as users shouldn't see it when opening docs.rs